### PR TITLE
add import_processes script

### PR DESCRIPTION
**Pull request by der-john**
this is a new version of #11 without the commits from master, missing one commit:

60d332b use total data dump.

### CODE REVIEW

File: meinberlin/management/commands/import_processes.py [hunk 1]
```diff
@@ -15,14 +16,13 @@
     'adhocracy_meinberlin.resources.burgerhaushalt.IProcess': (4, ''),
     'adhocracy_meinberlin.resources.stadtforum.IProcess': (5, ''),
     'adhocracy_meinberlin.resources.bplan.IProcess': (
-        1, 'SenStadtUm'),
+        6, 'SenStadtUm'),
```
Comment (der-john) [inline meinberlin/management/commands/import_processes.py]: thanks. I just wanted to push that.

File: meinberlin/management/commands/import_processes.py [hunk 2]
```diff
@@ -9,14 +9,24 @@
...
+RICOLLABORATIVE = 'adhocracy_meinberlin.resources.collaborative_text.IProcess'
+RIKIEZKASSE = 'adhocracy_meinberlin.resources.kiezkassen.IProcess'
+RIPROPOSAL = 'adhocracy_core.resources.proposal.IProposalVersion'
+RISTADTFORUM = 'adhocracy_meinberlin.resources.stadtforum.IProcess'
+SIDESCRIPTION = 'adhocracy_core.sheets.description.IDescription'
+SINAME = 'adhocracy_core.sheets.name.IName'
+SIPOOL = 'adhocracy_core.sheets.pool.IPool'
+SITITLE = 'adhocracy_core.sheets.title.ITitle'
+SIWORKFLOW = 'adhocracy_core.sheets.workflow.IWorkflowAssignment'
+
```
Comment (der-john) [inline meinberlin/management/commands/import_processes.py:23]: yeah, that looks better ... I always knew my green thumb needed help...

File: meinberlin/templates/meinberlin/includes/adhocracy_tile.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 <a class="span3 box-process" href="{% pageurl process %}">
     <div class="img-wrapper">
         {% image process.image fill-640x480 %}
-        <p class="copyright">&#169; {{ process.image_copyright }}</p>
+        {% if process.image_copyright %}<p class="copyright">&#169; {{ process.image_copyright }}</p>{% endif %}
```
Comment (der-john) [inline meinberlin/templates/meinberlin/includes/adhocracy_tile.html:6]: thanks.

**Comment by xi** [comment]
There are still some issues. I added some FIXMEs in 9d6eb8b. I propose to merge this soon and fix these issues seperately.

File: meinberlin/management/commands/import_processes.py [hunk 4]
```diff
@@ -67,23 +67,24 @@ def create_process(process, parent_process=None):
...
+                    short_description = (
+                        "Machen Sie Vorschläge, um Politik und "
+                        "Verwaltung dabei zu unterstützen, die knappen "
+                        "Finanzen des Bezirks bedarfsgerecht einzusetzen."
+                        )
+                elif process['content_type'] == RIKIEZKASSE:
+                    short_description = (
+                        "Hier können Sie Ihre Ideen und Vorschläge für die "
+                        "Kiezkasse abgeben."
+                        )
```
Comment (der-john) [inline meinberlin/management/commands/import_processes.py:90]: much better.

**Pull request merged**